receiptui/src/main/java/com/hyperwallet/android/ui/receipt/view/TabbedListReceiptsFragment.java

```diff
@@ -0,0 +1,235 @@
+
+    private TabbedListReceiptsViewModel mTabbedListReceiptsViewModel;
+
+    public TabbedListReceiptsFragment() {
+    }
+
+    public static TabbedListReceiptsFragment newInstance() {
+        TabbedListReceiptsFragment fragment = new TabbedListReceiptsFragment();
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+            @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_tabbed_list_receipt, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        mViewPager = view.findViewById(R.id.receipts_pager);
+        mTabLayout = view.findViewById(R.id.tab_layout);
+        mHeader = view.findViewById(R.id.transactions_header);
+    }
+
+    void retry() {
+        // mReceiptViewModel.retry();
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        if (getActivity() instanceof TabbedListReceiptsActivity) {
+            mHeader.setVisibility(View.GONE);
+        }
+        mTabbedListReceiptsViewModel = ViewModelProviders.of(this,
+                new TabbedListReceiptsViewModel.TabbedListReceiptsViewModelFactory(new UserRepositoryImpl(),
+                        new PrepaidCardRepositoryImpl()))
+                .get(TabbedListReceiptsViewModel.class);
+        mListReceiptsViewPagerAdapter = new ListReceiptsViewPagerAdapter(getFragmentManager(),
+                FragmentStatePagerAdapter.BEHAVIOR_RESUME_ONLY_CURRENT_FRAGMENT, new ArrayList<Parcelable>(),
+                getResources());
+        mListReceiptsViewPagerAdapter.addInitialItem(null);
+        mViewPager.setAdapter(mListReceiptsViewPagerAdapter);
+        mTabLayout.setupWithViewPager(mViewPager);
+        mTabbedListReceiptsViewModel.initialize();
+        registerObservers();
+    }
+
+    private void registerObservers() {
+        mTabbedListReceiptsViewModel.mPrepaidCards.observe(this, new Observer<List<PrepaidCard>>() {
+            @Override
+            public void onChanged(List<PrepaidCard> prepaidCards) {
+                if (!prepaidCards.isEmpty()) {
+                    boolean isWalletModel =
+                            mTabbedListReceiptsViewModel.mProgramModel.name() == ProgramModel.WALLET_MODEL.name();
+                    boolean isCardModel =
+                            mTabbedListReceiptsViewModel.mProgramModel.name() == ProgramModel.PAY2CARD_MODEL.name()
+                                    || mTabbedListReceiptsViewModel.mProgramModel.name()
+                                    == ProgramModel.CARD_ONLY_MODEL.name();
```

Comment:
this logic can be moved to ProgramModel, refer to same comment in create transfer PR